File: CS50 intro/problems/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height <1  or height > 8:
    height = get_int("Height: ")

for rows in range(height):
        print(" "*(height - rows -1) + ((rows)*"#"+"#" + "  " + (rows)*"#"+"#" ))

File: CS50 intro/problems/mario.c
#include <stdio.h>
#include <cs50.h>

//including note to compiler of function later in program
void print (char c, int n);
int main (void)


// The function itself
{
    //get int value from 1 - 8
    int n;
    do
    {
        n = get_int("Please enter a height for the pyramid: ");
    }
    while (n <1 || n> 8);

    //calling the first loop
    //note the loop function in a loop
    for (int i = 0;  i < n; i++)
    {
        print(' ', 7-i);
        print('#', i+1);
        print(' ', 2);
        print('#', i+1);     
        
        //adding a line break at the end
        printf("\n");
    }

}
// Defining the function, printing in a loop
void print(char c, int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("%c", c);
    }
}